.github/workflows/kind-e2e.yaml

```diff
@@ -17,8 +17,8 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-          - v1.31.6
-          - v1.32.2
+          - v1.32.x
+          - v1.33.x
```

Comment:
This won't work for Eventhing repo. The setup is not through chainguard/kind wrapper action, but rather kind's combination of exact version + image sha, see below `include:`.

Reply:
Yeah, I realized that after I pushed.  Fixed.